data/ecosystems/m/machinefi.toml

```diff
@@ -60,6 +63,12 @@ url = "https://github.com/machinefi/psa-crypto-esp32"
 [[repo]]
 url = "https://github.com/machinefi/smartpoint-demo"
 
+[[repo]]
+url = "https://github.com/machinefi/sprout"
+
+[[repo]]
+url = "https://github.com/machinefi/ssi-embedded"
```

Comment:
either remove or add missing label plz, getting 404

Reply:
updated! Thank you @eherrerosj - I didn't realize it was a private one